### ISSUE TITLE
Can you provide the training file?

### ISSUE THREAD

**Issue by PoonKinWang** [d-li14/dgconv.pytorch#4]
Can you provide the training file? Or some details of training setting?

**Comment by d-li14** [comment]
refer to #1

**Comment by PoonKinWang** [comment]
Thanks for your reply. Did you follow the parameters setting of the original paper? I have trained twice but didn't reach your accuracy.

**Comment by d-li14** [comment]
@PoonKinWang I train for 120 epochs with batch size 256 on 8 GPUs, initial LR 0.1, cosine LR decay, optimizer's weight decay and momentum following the official PyTorch ResNet training. 
Actually, the constraint is not used in my training. Just treat its training like a vanilla ResNet-50.

By the way, may I know what is the accuracy you achieved?

**Comment by PoonKinWang** [comment]
> @PoonKinWang I train for 120 epochs with batch size 256 on 8 GPUs, initial LR 0.1, cosine LR decay, optimizer's weight decay and momentum following the official PyTorch ResNet training.
> Actually, the constraint is not used in my training. Just treat its training like a vanilla ResNet-50.
> 
> By the way, may I know what is the accuracy you achieved?

I got 78.1% and 78.2% respectively. I to train for 90 epochs with batch size of 512 on 8 GPUs using distributed data parallel, and other setting like LR decay and hyper parameters are followed original paper as well. I am going to follow your setting to retrain. 
Thanks so much for your help.

**Comment by d-li14** [comment]
@PoonKinWang Yes, it will be very likely reproducible. Your result is only 0.5% lower on ImageNet.
Training longer will help a lot, so the gap will be filled up (the original paper seems not to specify training epochs). In addition, 32 samples per GPU is a good practice for ImageNet training, so the author uses batch size 512 on 16 GPUs.
